pulsar-broker/src/test/java/org/apache/pulsar/client/api/OrphanPersistentTopicTest.java

```diff
@@ -108,6 +114,67 @@ public void testNoOrphanTopicAfterCreateTimeout() throws Exception {
         pulsar.getConfig().setTopicLoadTimeoutSeconds(originalTopicLoadTimeoutSeconds);
     }
 
+    @Test
+    public void testCloseLedgerThatTopicAfterCreateTimeout() throws Exception {
```

Comment:
@shibd I tried this test without this fix. The test can still get passed. Do you have a test to reproduce the issue?

Reply:
Interesting. Are you copying the tests to the master branch to test it?

If use this PR branch and revert that fix code, it always fails.

Reply:
@shibd Yes, I just copied the tests to the master branch.

Reply:
Thanks reply, I changed this test, it should always fail if without this fix.